[2024-06-18T17:16:15.187+0545] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T17:16:15.213+0545] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: database_backup.send_mail scheduled__2024-06-18T11:20:00+00:00 [queued]>
[2024-06-18T17:16:15.220+0545] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: database_backup.send_mail scheduled__2024-06-18T11:20:00+00:00 [queued]>
[2024-06-18T17:16:15.220+0545] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-18T17:16:15.243+0545] {taskinstance.py:2330} INFO - Executing <Task(EmailOperator): send_mail> on 2024-06-18 11:20:00+00:00
[2024-06-18T17:16:15.246+0545] {standard_task_runner.py:63} INFO - Started process 4249 to run task
[2024-06-18T17:16:15.253+0545] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'database_backup', 'send_mail', 'scheduled__2024-06-18T11:20:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/backup.py', '--cfg-path', '/tmp/tmpxbfounyo']
[2024-06-18T17:16:15.254+0545] {standard_task_runner.py:91} INFO - Job 21: Subtask send_mail
[2024-06-18T17:16:15.354+0545] {task_command.py:426} INFO - Running <TaskInstance: database_backup.send_mail scheduled__2024-06-18T11:20:00+00:00 [running]> on host PRABAL-10.
[2024-06-18T17:16:15.488+0545] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='prabal' AIRFLOW_CTX_DAG_ID='database_backup' AIRFLOW_CTX_TASK_ID='send_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T11:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T11:20:00+00:00'
[2024-06-18T17:16:15.490+0545] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T17:16:15.514+0545] {logging_mixin.py:188} WARNING - /home/prabal/afenv/lib/python3.10/site-packages/airflow/utils/email.py:154 RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
[2024-06-18T17:16:15.515+0545] {email.py:271} INFO - Email alerting: attempt 1
[2024-06-18T17:16:18.332+0545] {email.py:282} INFO - Sent an alert email to ['susmapant@gmail.com']
[2024-06-18T17:16:20.543+0545] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T17:16:20.552+0545] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=database_backup, task_id=send_mail, run_id=scheduled__2024-06-18T11:20:00+00:00, execution_date=20240618T112000, start_date=20240618T113115, end_date=20240618T113120
[2024-06-18T17:16:20.601+0545] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-18T17:16:20.622+0545] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-18T17:16:20.623+0545] {local_task_job_runner.py:222} INFO - ::endgroup::
